app/packages/_components/list_flank/list_flank.coffee

```diff
@@ -15,16 +27,37 @@ Template.list_flank.events
     else
       data.selected.set itemId
 
+    _allItemsSelected = instance.allItemsSelected
+    if allItemsSelected(instance)
```

Comment:
good call. I suppose we could do something like this ```_allItemsSeleted.set allItemsSelected(instance)``` since the function returns a boolean?
Do you think the naming is confusing?

We should also, in this PR or in the future, resolve the issue in the comment. I think we could move some code in a template then pass stuff down. That way we will have direct access to the current collection.